### ISSUE TITLE
HTML5 export issue with get_executable_path

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#3569]
When running the 2.0-beta HTML5 template, I get (in Firefox, safe mode to prevent my ad blocker from letting hell break loose):

```
let it go!
[1;31mERROR: get_executable_path: [0m[1mWarning, don't know how to obtain executable path on this OS! Please override this function properly.
[0;31m   At: drivers/unix/os_unix.cpp:508.[0m
```

I'm not sure why it tries to use drivers/unix when it's the javascript platform, that might be a first clue. My test OS is indeed a Linux 64-bit though.

**Comment by castled** [comment]
Seeing the same on Windows/Chrome. Used the HTML5 template included in the 2.0.1 build downloaded from Steam.

I'm a godot noob, would like to know if I'm doing something wrong!

**Comment by Reparadocs** [comment]
Getting the same issue as well

**Comment by Windigo** [comment]
Same here on Mac

**Comment by vnen** [comment]
Isn't this just a matter of overriding `get_executable_path()` in `OS_Javascript`? Not sure what it's supposed to return in this case (maybe the current URL?).

**Comment by eska014** [comment]
This was fixed in #4382
